PatientRegistrationApp/src/components/SinglePatientEmergencyContact.vue

```diff
@@ -37,12 +43,36 @@
 </template>
 
 <script>
+import EventBus from '../eventBus'
+
 export default {
   name: "SinglePatientEmergencyContact",
   data() {
     return {
       phoneType: ["Cell", "Home"],
+      contactFamilyName: '',
+      contactGivenName: '',
+      contactPhoneNumber: '',
+      contactPhoneNumberType: ''
```

Comment:
Would be good to consider the eventbus setup for a household where there might be multiple emergency contacts, i.e. should the naming convention change for this?

Reply:
We can use the same strategy (event bus), but the data will be forwarded to a separate page (review and submit page for the "household" registration).  We will still have only 1 emergency contact entry for the household registration (for patient #1), but then patient #1 will be the emergency contact for everyone else in the household.